docs/documentation/development/bindings/thing-definition.md

```diff
@@ -249,12 +292,57 @@ Depending on the solution the provided meta data can be used for different purpo
     </thing-type>
 ```
 
-In general each `property` must have a name attribute which should be written in camel case syntax. The actual property value is defined as plain text and is placed as child node of the property element. It is recommended that at least the vendor and the model id properties are specified here since they should be definable for the most of the devices. In contrast to the properties defined in the 'ThingType' definitions the thing handler [documentation](thing-handler.html) explains how properties can be set during runtime.
+In general each `property` must have a name attribute which should be written in camel case syntax. 
+The actual property value is defined as plain text and is placed as child node of the property element. 
+It is recommended that at least the vendor and the model id properties are specified here since they should be definable for the most of the devices. 
+In contrast to the properties defined in the 'ThingType' definitions the thing handler [documentation](thing-handler.html) explains how properties can be set during runtime.
+
+### Representation Property
+
+A thing type can contain a so-called `representation property`. 
+This optional property contains the _name_ of a property whose value can be used to uniquely identify a deivce. 
+The `thingUID` cannot be used for this purpose because there can be more than one thing for the same device. 
+
+Each physical device normally has some kind of a technical identifier which is unique.
+This could be a MAC address (e.g. Hue bridge, camera, all IP-based devices), a unique device id (e.g. a Hue lamp) or some other property that is unique per device type. 
+This property is normally part of a discovery result for that specific thing type. 
+Having this property identified per binding it could be used as the `representation property` for this thing.
+
+The `representation property` will be defined in the thing type XML: 
+
+```xml 
+ <bridge-type id="thingTypeId">
```

Comment:
Well done. I recently asked about  the meaning of this so called representation property.

Just a minor question: We have an opening `bridge-type`-tag in this example and a closing `thing-type`-tag. It maybe result in confusing people. Doesn't it?

Reply:
agreed. bug fix PR appreciated :-)  (should be both `thing-type`)

Reply:
PR #4355